airbyte-integrations/connectors/source-zoom/source_zoom/schemas/meeting_poll_results.json

```diff
@@ -5,12 +5,21 @@
     "meeting_uuid": {
       "type": "string"
     },
+    "meeting_id": {
+      "type": "integer"
```

Comment:
The schema properties should be `nullable`, a.k.a: `type: ["null", "integer"]`. Everywhere.

Reply:
Done and done